### DISCUSSION TITLE
How about adding some highlighting system?

### DISCUSSION THREAD

**Discussion by ovizii** [karakeep-app/karakeep#77] [Ideas]
I've seen a couple of read-later apps offering that, and I kinda like to highlight the most important parts of an article I bookmark for revising later.

**Comment by parthpat12** [comment]
Yes! This needs more attention. Would really like to make the switch to hoarder from Raindrop but one thing this is missing is this highlight feature.

**Comment by kamtschatka** [comment]
are we talking about notes you make (where you can already use markdown) or are you talking about highlighting something in the cached content of a link bookmark?

**Reply by ovizii** [reply]
I was talking about highlighting something in the cached content of a link bookmark.

**Comment by ChristophCan** [comment]
I think this feature is already implemented by now, as you can see from the features List: 

![2025-02-10_13-42](https://github.com/user-attachments/assets/624c61e3-51b7-4f90-a0c4-70a71fe3c183)
